src/components/CurratedContentCard/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Image } from '../Image'
+import {
+  color,
+  radius,
+  space,
+  fontSize,
+  fontWeight,
+  device,
+  lineHeight,
+} from '../../theme'
+import { SmallText } from '../Text'
+import { transition } from '../../../src'
+
+const Container = styled.div`
+  background-color: ${color.elevatedBackground};
+  display: flex;
+  inline-size: 302px;
+  block-size: 314px;
+  flex-direction: column;
+  cursor: pointer;
+  border-radius: ${radius.lg};
+  overflow: hidden;
+
+  @media ${device.tablet} {
+    inline-size: 639px;
+    block-size: 244px;
+    flex-direction: row;
+  }
+
+  @media ${device.laptop} {
+    &:hover {
+      img {
+        transition: all ${transition};
```

Comment:
Do you need to transition all? Maybe better to transition only the `scale`?

Reply:
yeah it makes it smoother